scripting/weaponmodels.sp

```diff
@@ -381,7 +432,7 @@ public Action OnWeaponSwitch(int client, int weapon)
 		}
 		else if (g_WeaponModelInfo[i][WeaponModelInfo_Forward])
 		{
-			if (!ExecuteForward(i, client, weapon, className))
+			if (!ExecuteForward(i, client, weapon, className, -1))
```

Comment:
what is the added param doing?

Reply:
New sourcemod syntax does not support default function parameter values. Therefore every function call that had used a default val needed the default val added in its params .